packages/components/src/ExternalLink/ExternalLink.styles.js

```diff
@@ -0,0 +1,6 @@
+import { css } from '@wp-g2/styles';
+
+export const Icon = css`
+	fill: currentColor;
+	margin-left: 0.1em;
```

Comment:
I'm not too sure what `0.1em` would compute to 🤔 .

If it's something close to a spacing system value, would be be able to try the `ui.space()` util 🎉 
https://github.com/ItsJonQ/g2/blob/master/packages/styles/types/presets/space.d.ts#L14